packages/commonwealth/scripts/wait-server.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+attempt_counter=0
```

Comment:
Can you write a comment at the top of this script explaining what it does?

Reply:
I have created the description here along with a loom example:
https://github.com/hicommonwealth/commonwealth/wiki/Package.json-Scripts